api/tests/functional-tests/backend/core/test_dataset.py

```diff
@@ -104,31 +15,31 @@ def created_datasets(db: Session) -> list[str]:
     return ["dataset1", "dataset2"]
 
 
-def test_create_dataset(db: Session, created_dataset):
+def test_create_dataset(db: Session, core_created_dataset):
     dataset = db.query(
         select(models.Dataset)
-        .where(models.Dataset.name == created_dataset)
+        .where(models.Dataset.name == core_created_dataset)
         .subquery()
     ).one_or_none()
     assert dataset is not None
-    assert dataset.name == created_dataset
+    assert dataset.name == core_created_dataset
```

Comment:
whats the reason for this name change from `created_dataset` to `core_created_dataset`?

Reply:
I was worried about fixture name conflict in conftest.py

Reply:
can you do the rename in a separate PR?